.travis.yml

```diff
@@ -0,0 +1,32 @@
+---
+language: node_js
+node_js:
+  - "8"
+
+sudo: false
+dist: trusty
+
+branches:
+  only:
+    - master
+
+matrix:
+  include:
+  - name: "npm"
+    before_install:
+      - npm config set spin false
```

Comment:
is this still a thing with npm?! 😱

Reply:
tbh I think that Travis defaults to `npm ci` now so no 😂

Reply:
but we're not using that anywhere, are we?

Reply:
yes that's true, it's part of the `npm link`. It seems to work fine without needing this line 👍